### PR TITLE
Transaction ckBTC simplify estimation label

### PR DESCRIPTION
**Pull request by peterpeterparker**
# Motivation

Instead of repeating "Estimated..." on review step, display a notice.

# Changes

- update label and display notice

# Screenshots

Before:

Capture d’écran 2023-04-15 à 08 30 23

After:

Capture d’écran 2023-04-15 à 08 42 13

### CODE REVIEW

File: frontend/src/lib/types/i18n.d.ts [hunk 1]
```diff
@@ -238,13 +238,16 @@ interface I18nAccounts {
   network_btc_mainnet: string;
   network_btc_testnet: string;
   select_network: string;
+  bitcoin_transaction_fee_notice: string;
   estimated_bitcoin_transaction_fee: string;
```
Comment (lmuntaner) [inline frontend/src/lib/types/i18n.d.ts:242]: Is this key still used?

File: frontend/src/lib/types/i18n.d.ts [hunk 2]
```diff
@@ -238,13 +238,16 @@ interface I18nAccounts {
...
+  internetwork_fee_notice: string;
   estimated_internetwork_fee: string;
```
Comment (lmuntaner) [inline frontend/src/lib/types/i18n.d.ts:244]: Is this still used?

**Review by lmuntaner — APPROVED** [review]
Minor comments, already approving.

File: frontend/src/lib/types/i18n.d.ts (hunk 1 above)
Comment (peterpeterparker) [reply frontend/src/lib/types/i18n.d.ts:242]: Yes

**Pull request merged**